packages/klighd-core/src/views-rendering.tsx

```diff
@@ -518,6 +568,43 @@ export function renderKText(rendering: KText, parent: SKGraphElement | SKLabel,
             attrs.textLength = rendering.calculatedTextLineWidths[0]
             attrs.lengthAdjust = 'spacingAndGlyphs'
         }
+
+        if (context.depthMap) {
+            if (boundsAndTransformation.bounds.width && boundsAndTransformation.bounds.height
+                && (rendering.isNodeTitle || (text !== '-' && text !== '+'))) {
```

Comment:
This seems to be a hack to avoid some behavior that happens with SCCharts, this code however should be generic for any synthesis. Is there a way to avoid this?

Reply:
I don't know how to correctly identify the text elements we want to handle here and as is it appears to have plenty of false positives.

What we would want is all Node Titles and all Region Titles, the former appears to be covered correctly, but I don't know how to identify the latter.

Reply:
My latest commits here in combination with the enabling commits to klighd and semantics should resolve this

Reply:
as the former turned out to be not necessary and the later has been merged this should by now be resolved